### PR TITLE
Fix  HTML in template file + Fix languages for typo and language keys in too

### PR DESCRIPTION
**Pull request by Galixte**
Hi @vinny & @dmzx,

to fix this issue: https://github.com/dmzx/youtubegallery/issues/20.

### CODE REVIEW

**Comment by vinny** [comment]
@dmzx what do you think?

**Comment by Galixte** [comment]
hI @dmzx,

could you merge this, please ?

**Comment by dmzx** [comment]
hard coded lang edits wont be allowed by phpbb.com
https://github.com/Galixte/youtubegallery/commit/3544fec8f1ce7921db1fbdbbfc6521f41b67ed7a#diff-22bb76bbe2ddd8e46458677a4375aeabR43

**Comment by Galixte** [comment]
You talking about: “youtube.com” & “youtube.be” ?

**Comment by Galixte** [comment]
@dmzx, i think it's OK, now.

**Comment by vinny** [comment]
Looks fine to me. @dmzx

File: styles/prosilver/template/video_editor.html [hunk 1]
```diff
@@ -37,7 +37,13 @@
 	<div class="rules">
 		<div class="inner">
 			<strong>{L_VIDEOS_TIP}</strong><br />
-			{L_VIDEOS_TIPS}
+				<ul>
+					<li>{L_VIDEOS_TIPS_PART_01}&nbsp;<a href="https://www.youtube.com/">Youtube.com</a>,&nbsp;{L_VIDEOS_TIPS_PART_02}</li>
```
Comment (vinny) [inline styles/prosilver/template/video_editor.html]: replace Youtube.com with {L_VIDEOS_TIPS_PART_05}

File: language/en/common.php [hunk 2]
```diff
@@ -24,15 +44,15 @@
...
-	</ul>
-	<br />
-	<strong>Warning: this page isn’t for uploading videos to Youtube!</strong>',
-	'UNAUTHED'				=> 'You are not authorised to view this page.',
+	'VIDEOS_TIPS_PART_01'			=> 'Browse to',
+	'VIDEOS_TIPS_PART_02'			=> 'search your favorite videos.',
+	'VIDEOS_TIPS_PART_03'			=> 'Copy the video URL, paste it in the field above, choose the category and submit the form.',
+	'VIDEOS_TIPS_PART_04'			=> 'You can use',
+	'VIDEOS_TIPS_PART_05'			=> 'youtube.com',
+	'VIDEOS_TIPS_PART_06'			=> 'youtube.be',
```
Comment (vinny) [inline language/en/common.php]: rename to <code>youtu.be</code>

File: language/fr/common.php [hunk 3]
```diff
@@ -31,74 +35,72 @@
...
-	<strong>Avertissement: cette page ne peut pas être utilisée pour charger des vidéos sur Youtube!</strong>',
-	'UNAUTHED'				=> 'Vous n’êtes pas autorisé à visualiser cette page.',
-	'VIDEO_UNAUTHED'		=> 'Vous n’êtes pas autorisé à visualiser cette vidéo.',
-	'INVALID_VIDEO'			=> 'La vidéo que vous avez sélectionné n’existe pas.',
+	'VIDEOS_TIPS_PART_01'			=> 'Se rendre sur',
+	'VIDEOS_TIPS_PART_02'			=> 'pour trouver ses vidéos préférées.',
+	'VIDEOS_TIPS_PART_03'			=> 'Copier l’adresse URL de la vidéo, la coller dans le champ ci-dessous, choisir la catégorie souhaitée et valider le formulaire.',
+	'VIDEOS_TIPS_PART_04'			=> 'Il est possible d’utiliser',
+	'VIDEOS_TIPS_PART_05'			=> 'youtube.com',
+	'VIDEOS_TIPS_PART_06'			=> 'youtube.be',
```
Comment (vinny) [inline language/fr/common.php]: rename to <code>youtu.be</code>

File: language/ru/common.php [hunk 4]
```diff
@@ -25,15 +45,15 @@
...
-	</ul>
-	<br />
-	<strong>Внимание: Эта страница не для загрузки видеороликов с Youtube!</strong>',
-	'UNAUTHED'				=> 'Вы не авторизованы для просмотра этой страницы.',
+	'VIDEOS_TIPS_PART_01'			=> 'Перейдите на',
+	'VIDEOS_TIPS_PART_02'			=> ' и выберите видеоролик который Вам понравился.',
+	'VIDEOS_TIPS_PART_03'			=> 'Скопируйте URL видеоролика, вставьте его в поле выше и нажмите отправить.',
+	'VIDEOS_TIPS_PART_04'			=> 'Ссылки принимаются с',
+	'VIDEOS_TIPS_PART_05'			=> 'youtube.com',
+	'VIDEOS_TIPS_PART_06'			=> 'youtube.be',
```
Comment (vinny) [inline language/ru/common.php]: rename to <code>youtu.be</code>

**Comment by vinny** [comment]
Okay, I think that we can use this:
```
	'VIDEOS_TIPS_PART_01'			=> 'Browse to <a href="https://www.youtube.com/">Youtube.com</a>, search your favorite videos.',
	'VIDEOS_TIPS_PART_02'			=> 'Copy the video URL, paste it in the field above, choose the category and submit the form.',
	'VIDEOS_TIPS_PART_03'			=> 'You can use <strong>youtube.com</strong> and <strong>youtu.be</strong>,both are accepted by the Extension.',
	'VIDEOS_TIPS_PART_04'			=> 'Warning',
	'VIDEOS_TIPS_PART_05'			=> 'this page isn’t for uploading videos to Youtube!',

**Comment by dmzx** [comment]
yes lets go for this one, new pull request?

**Comment by Galixte** [comment]
No, i will update this one tonight, thanks for feedbacks.

**Comment by Galixte** [comment]
@dmzx , Done.

**Comment by Galixte** [comment]
Ok, you’re right, it’s better with new PR, here: https://github.com/dmzx/youtubegallery/pull/26.

**Pull request closed without merging**